src/components/__tests__/__snapshots__/StackedAreaChart.spec.js.snap

```diff
@@ -12,7 +12,7 @@ exports[`component: StackedAreaChart initial state renders without crashing 1`]
     className="m-notification_content"
```

Comment:
Do a find and replace for `m-notification_content` and update it to `m-notification__content`

I wonder if you ran the find and replace script again whether it would catch these on a second go 🤔

Reply:
I tried and it did not find anything